tests/settings/ConfigurationGatheringTest.py

```diff
@@ -81,13 +81,51 @@ def test_default_coafile_parsing(self):
 
         Constants.system_coafile = tmp
 
+    def test_capture_warnings_for_default_coafile(self):
+        """
+        Since this addresses the deprecated method, this testcase is
+        temporary (until the old API is fully removed).
+        """
+        tmp = Constants.system_coafile
+
+        Constants.system_coafile = os.path.abspath(os.path.join(
+            os.path.dirname(os.path.realpath(__file__)),
+            'section_manager_test_files',
+            'default_coafile'))
+
```

Comment:
will it work? Its not checking default_coafile inside section_manager_test_files in the main code